README.md

```diff
@@ -1,14 +1,207 @@
+
+The `year_column_check()` function has the following criteria, where all must be met for it to flag a column containing time period information (that is, year information):
+
+* is the length of the value 4 characters long?
+* is the variable class numeric, double or integer? 
+* are there no NA values in the column?
+* do the first characters of the values start with 19 or 20?
+
+Each pass equates to a value of 1. If the sum of all the checks is 4 (for the number of tests), then the column in question contains time period data.
+
+The argument required for the function is a column in a dataframe. In the context of the `tidied_df()` function, this is looped through all columns in a dataframe, so no input is required for the purposes on TiN.
+
+#### create_bq_table
+
+You can call this function using the following line:
+
+```
+tinScrape::create_bq_table()
+```
+
+This function exports the tidied data to BQ. It checks if the data table already exists in BQ so it can truncate/overwrite the data in it. It is set up this way to ensure the latest information is always reflected in BQ. Otherwise it will create a new table in the `dft-stats-diss-dev` dataset in BQ and populate it with the data.
+
+Arguments required are:
+
+* the name of tidied data
+* the name of the worksheet, found in the Excel document
+* the name of the Excel document
+
+Due to this function being part of the `gcp_to_bq()` function, the arguments stated above do not need to be manually inputted as this is automated.
+
+### scrape_dynamic_methodology
+
+You can call this function using the following line:
+
+```
+tinScrape::scrape_dynamic_methodology()
+```
+This function webscrapes the any technical notes that accompany a publication. This covers notes and definitions and background quality reports. The function calls on other functions that are found in the utils.R script which are designed to extract hyperlinks that pertain to statistical outputs only (specifically the `collect_collections()` and `collect_links()` functions), and regex-based functions, such as `grepl()`, to filter for specific words that relate to these types of documents.
+
+No argument is required to use this function as the Stats at DfT webpage is hardcoded into the function in the first instance.
+
+### scrape_orr_tables
+
+You can call this function using the following line:
+
+```
+tinScrape::scrape_orr_tables()
+```
+
+This function webscrapes the downloadable hyperlinks for published data tables from the Office for Rail and Road's (ORR) data portal, to obtain rail data published by the organisation. The function calls the `extract_orr_pages()` function, found in the utils.R script, to extract hyperlinks that pertain to statistical outputs only.
+
+No argument is required to use this function as the ORR's passenger rail usage webpage is hardcoded into the function in the first instance. This is where the rail data of current interest is hosted on the site.
+
+### utils
```

Comment:
I would probably drop this section entirely

Reply:
I'm of the mind to keep it in, or have it somewhere, in case volunteers/new joiners need to know what these functions do, so there's at least a written record somewhere

Reply:
Fair, possibly just rename the section as "utility functions" then?